mvc/models.js

```diff
@@ -55,3 +56,13 @@ exports.getCommentsByArticleIdModel = (articleId) => {
             return rows
         })
 }
+
+exports.postCommentModel = ({username, body}, articleId) =>{
+    const commentValues = [body, 0, username, articleId]
+      return db
+      .query(`INSERT INTO comments (body, votes, author, article_id, created_at) VALUES ($1, $2, $3, $4, CURRENT_TIMESTAMP) RETURNING *;`, commentValues)
+      .then(({rows})=>{
+          return rows[0]
+      })
```

Comment:
Very tidy query. If you look at the seed you'll see the votes is defaulted to 0 anyway so you wouldn't get an error if you omitted it.